src/main/java/frc/robot/flywheel/FlywheelSubsystem.java

```diff
@@ -14,33 +18,42 @@ public class FlywheelSubsystem extends SubsystemBase {
      * Total: 2 TalonFX motors
      */
 
+    private final TalonFX masterFlywheel;
+    private final TalonFX followerFlywheel;
+
     public FlywheelSubsystem() {
+        masterFlywheel = new TalonFX(FlywheelConstants.LEFT_CAN_ID);
+        followerFlywheel = new TalonFX(FlywheelConstants.RIGHT_CAN_ID);
+
+        followerFlywheel.follow(masterFlywheel);
     }
 
     /*
      * Stop the flywheel from moving
      */
     public void stopFlywheel() {
+        masterFlywheel.set(ControlMode.PercentOutput, 0);
```

Comment:
You should be using motor.neutralOutput() instead of setting percent to 0 because the flywheel will try to PID to zero which is unnecessary. neutralOutput() just stops giving power to the motor so it slowly comes to a stop.